accounts/models.py

```diff
@@ -1,6 +1,10 @@
 from django.conf import settings
 from django.db import models
 from django.contrib.auth.models import AbstractBaseUser, BaseUserManager
+from decimal import Decimal
+import numpy as np
+
+DEFAULT_LOAN_AMOUNT = getattr(settings, 'DEFAULT_LOAN_AMOUNT', Decimal(12000.00))
```

Comment:
The bank model hasn't been created yet so is this line of code necessary?

Reply:
Yes, it is required in User Model